platform/jobs/pipelines/edxPlatformQualityPipelinePr.groovy

```diff
@@ -0,0 +1,78 @@
+package platform
+
+import org.yaml.snakeyaml.Yaml
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+
+/* stdout logger */
+Map config = [:]
+Binding bindings = getBinding()
+config.putAll(bindings.getVariables())
+PrintStream out = config['out']
```

Comment:
^ this is only used for creating a logger to make debugging easier when parsing yml files

Reply:
nevermind